src/data/cond.ml

```diff
@@ -64,9 +64,9 @@ module Operation = struct
   let false_ = Value Objt.false_
   let and_ x y = op2 x Op.And_ y
   let and_s x y =
-    if x = Value (Objt.true_) then
+    if Stdlib.(x = Value) (Objt.true_) then
```

Comment:
This does not compile on my machine, I fixed it with

```diff
- if Stdlib.(x = Value) (Objt.true_) then
+ if Stdlib.(x = Value Objt.true_) then
```

My fix seems fine to me, but you said you compiled the project and I don't understand how given this line.